### ISSUE TITLE
Does this work with the Tascam us-1800? 

### ISSUE THREAD

**Issue by CircularTonic** [onkelDead/tascam-gtk#31]
Been trying to install this on my system to see if it works with my unsupported Tascam us-1800. I seen some people supposedly get it to work. I don't want to give up on this audio interface. It's becoming "vintage" now though.

**Comment by oddtime** [comment]
If it can help, the kernel driver for Tascam us16x08 (recompiled after some minor modifications in the code, such as changing the usb id for the different model of the soundcard) is working for a tascam us20x20. Of course there are some differences (number of channels and other things) but the majority of controls seem to work.

**Comment by onkelDead** [comment]
@oddtime Sounds like it makes sense to introduce a separated quirk on it. Any plans?

**Comment by oddtime** [comment]
@onkelDead yes, I am working on it (for the us20x20). Hopefully I will integrate soon the additional parameter controls (reverb, aux sends...). I will let you know as soon as it is fully working and so we can extend the very same gui and kernel driver to control both tascam interfaces if you want. At the moment I have duplicated the files for the experiments and I am not very experienced.

**Comment by oddtime** [comment]
Wonderful! I am on lubuntu

**Comment by nicola-lunghi** [comment]
hi @onkelDead @oddtime  i have an us-20x20 as well

**Comment by oddtime** [comment]
Good! I have managed to add the "aux" and "reverb" controls in the kernel driver...

**Comment by nicola-lunghi** [comment]
> Good! I have managed to add the "aux" and "reverb" controls in the kernel driver...

I haven't tried it yet with linux as at the moment I had to get space from my windows partition.
I'll check if it works would be nice to use it also there.

Question: does the device allows access to the output routing?

**Comment by oddtime** [comment]
> Question: does the device allows access to the output routing?

Yes
